starter-code/css/main.css

```diff
@@ -6,3 +6,140 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+    font: 200 20px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+    margin:0;
+    padding:0;
+
+}
+
+.margin{
+    margin-left: 20px;
+    margin-right: 20px;
+}
+
+
+nav {
+    padding: 0;
+    background: white;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    justify-content: space-between ;
+}
+
+.bg {
+    background-image: url(../images/landing.jpg);
+    background-position: center; 
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 600px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    color: white;
+    padding: 0 200px;
+    flex-direction: column;
+    
+}
+
+h1 {
+    font-size: 70px;
+    margin:0;
+}
+
+h3 {
+    font-size: 40px;
+    margin: 0;
+}
+
+.section2 {
+    text-align: center;
+    margin: 50px 100px;
+}
+
+li{
+    margin-left: 10px;
+    margin-right: 10px;
+    display: flex;
+    list-style-type: none;
+    padding: 0;
+    float: right;
+}
+
+a {
+    color: #1A1A1A;
+    text-decoration: none;
+}
+
+
+.bg-green {
+    background-color: #00B172;
+    background-image: url(../images/spotify-icon-white.png);
+    background-position: center; 
+    background-repeat: no-repeat;
+    background-size: 10%;
+    color: white;
+    justify-content: space-around;
+    padding: 70px;
+}
+
+.flex {
+    display: flex; 
+}
+
+.center {
+    align-items: center;
+}
+
+.column {
+    flex-direction: column;
+    margin-top: 100px;
+    margin-bottom: 100px;
+    width: 33%;
+    margin-left: 50px;
+    margin-right: 50px;
+
+  
+}
+
+.spotify-logo {
+    float: left;
+    height: 50px;
+}
+
+.icons {
+
+    height: 60px;
+
+}
+
+#spot-app {
+    width: 300px;
+
+}
+
+.p_mr_150 {
```

Comment:
Some of your css names are a little confusing- try to give them meaningful names because it will help a lot when working on a group project. And remember the naming convention for css is kebab-case and not snake_case